Tests/ProjectTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Allure.Net.Commons;
+using Allure.NUnit.Attributes;
+using TMS_Selenium.Models;
+using TMS_Selenium.Pages.ProjectPages;
+using TMS_Selenium.Test;
+
+namespace TMS_Selenium.Tests
+{
+    [AllureEpic("TestRail")]
+    [AllureFeature("Basic Functionality")]
+    public class ProjectTests : BaseTest
+    {
+        [Test]
+        [AllureDescription("Verifying a project with name and type has been added")]
+        [AllureSeverity(SeverityLevel.critical)]
+        [AllureStory("Add a project")]
+        public void AddProjectWithNameAndType()
+        {
+            string projectName = "EAntonova_" + Guid.NewGuid();
+
+            userStep.SuccessfulLogin();
+            logger.Info("Login successful");

```

Comment:
you can hide loggers into methods
for example this logger.Info("Login successful"); you can move to userStep.SuccessfulLogin(); and your test method will be clean